src/typeahead/test/typeahead.spec.js

```diff
@@ -620,8 +636,6 @@ describe('typeahead tests', function () {
 
       // Note that this bug can only be found when element is in the document
       $document.find('body').append(element);
-      // Extra teardown for this spec
-      this.after(function () { element.remove(); });
```

Comment:
Yep, tests will always pass but this is a DOM leak, so if you keep the tests running a long time, the browser will crash.